client/web/compose/src/components/PageBlocks/Wrap/Card.vue

```diff
@@ -135,6 +135,14 @@ export default {
     footerSet () {
       return !!this.$scopedSlots.footer
     },
+
+    showHeaderItems () {
+      return this.headerSet || this.block.title || this.block.description || this.hasMagnifyIcon || this.block.options.refreshRate >= 5 || this.block.options.manualRefresh
```

Comment:
To avoid a long chain you could simplify this with [condition1, condition2].some(v => !!v)
Same for the below one